garrysmod/gamemodes/terrortown/gamemode/lang/english.lua

```diff
@@ -746,7 +746,7 @@ L.tip39 = "C4 that explodes due to a mistake in disarming it has a smaller explo
 L.tip40 = "If it says 'HASTE MODE' above the round timer, the round will at first be only a few minutes long, but with every death the available time increases (like capturing a point in TF2). This mode puts the pressure on the traitors to keep things moving."
 
 -- 9/22/21
-L.tip41 = "You can adjust a player's microphone volume by right-clicking their mute button at the end of the scoreboard."
+L.tip41 = "You can adjust a player's microphone volume by right-clicking on their mute icon in the scoreboard."

```

Comment:
> right-clicking their mute button

It is already perfectly valid English.

Reply:
For me, "the end of the scoreboard" is at the very bottom of it, like the end of the file.

Reply:
> For me, "the end of the scoreboard"

That's why I didn't quote that part. I am referring to the addition of `on`, which is redundant at best, and incorrect at worst.

Reply:
Oh, my bad. I didn't realize you referring to the `on`. Well, if it's really redundant in English, then I have nothing against removing it and changing it to something like:
```suggestion
L.tip41 = "You can adjust a player's microphone volume by right-clicking their mute icon in the scoreboard."
```